spec/models/knight_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+require 'knight'
+
+RSpec.describe Knight, type: :model do
+
+  before do
+    # Create test data example
+    @knight = Knight.create(:x_position => 3, :y_position => 3)
+  end
+
+  it "should know if x or y move is two then the next x or y move is one vice versa" do

```

Comment:
Probably no need for this if you refactor as above.

Try to test the overall logic more - place you piece on different positions and try to move to different places and see if you get true or false..